rskj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -1425,6 +1437,10 @@ private int getReturnDataBufferSizeI() {
         return Optional.of(copiedData);
     }
 
+    public BlockchainConfig getBlockchainConfig() {
+        return blockchainNetConfig.getConfigForBlock(getNumber().longValue());
```

Comment:
We call this method multiple times from `VM`, and it might be a performance bottleneck. I don't want to do premature optimization, but how about caching it in the constructor? The block number should never change during a program execution.